torch/testing/_internal/distributed/distributed_test.py

```diff
@@ -31,6 +31,7 @@
     default_hooks as default,
     quantization as quantization_hooks,
 )
+from torch.distributed.optim import _apply_optimizer_in_backward
```

Comment:
Do these added unit tests cover both `DDP_OVERLAPPED_OPTIM_SET_GRADS_TO_NONE = 1` and `0` cases? I am not seeing the `0` case.